docker/README.md

```diff
@@ -0,0 +1,35 @@
+This is a Docker container for building all the packages in the monorepo and providing their build artifacts. Other package specific containers can use this one to obtain files necessary for their run without any additional dependencies.
+
+# Build
```

Comment:
Maybe add a single hashtag title and use ## as subheaders for the other sections?